src/main/java/at/fhhagenberg/esd/sqe/ws20/gui/ECCController.java

```diff
@@ -383,18 +389,31 @@ protected void gotoTargetFloor(ActionEvent event) {
             log(Messages.getString("invalidSelection"));
     }
 
-    private void translateElevator(Integer percentage) {
+    private void translateElevator(Integer percentage, Boolean triggerResize) {
+    	if(triggerResize) {
```

Comment:
Well, that is ugly af. An overload without parameters which translates the elevator to `oldPercentage` would do, I guess.

Reply:
And btw, formatting!

Reply:
Why u always have so good architectural thoughts. Changed it accordingly. 

Formatted it now by hand hope it is any better?